packages/desktop-client/src/components/manager/subscribe/common.js

```diff
@@ -46,6 +45,30 @@ export function useBootstrapped() {
         } else {
           ensure('/bootstrap');
         }
+      };
+
+      let url = await send('get-server-url');
+      if (url == null) {
+        // A server hasn't been specified yet
+        try {
+          let serverURL = window.location.origin;
+          let { error, hasServer } = await send('subscribe-needs-bootstrap', {
+            url: serverURL,
+          });
```

Comment:
Seems a bit redundant to make this API call 2x (once here and once in `afterBootstrap`). WDYT?

Reply:
Good point. Reduced the 3 total calls down to 1.